sys/include/xtimer.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
  * @note This is a struct in order to make the xtimer API type strict
  */
 typedef struct {
-    uint64_t ticks64;
+    uint64_t ticks64;       /**< Captain Obvious to the rescue! */
```

Comment:
I am with @OlegHahm here, `Captain Obvious` is well underestimated and should get some recognition! Or are you a golfer? :-)